src/constructs/iam/roles/instance-role.ts

```diff
@@ -11,11 +11,13 @@ import {
 } from "../policies";
 import { GuRole } from "./roles";
 
-interface GuInstanceRoleProps extends AppIdentity {
+export interface GuInstanceRoleProps {
   withoutLogShipping?: boolean;
   additionalPolicies?: GuPolicy[];
 }
 
+export type GuInstanceRolePropsWithApp = GuInstanceRoleProps & AppIdentity;
```

Comment:
Hopefully we can work on [this ticket](https://trello.com/c/QxhYoo4c/654-refactor-abstract-away-the-complexities-of-being-app-aware-construct) soon so that I can stop doing this 🙈